### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by gspetro-NOAA**
### DESCRIPTION OF CHANGES:
Created HPC Stack Install doc
Making Introduction section more newbie-friendly and concise
Created Components doc to relocate detailed info from the Introduction
Updated Quickstart to include tips beyond Level 1 systems & to reflect HPC Stack install and installation by container
Also added a Quickstart section on staging data (for those not on a Level 1/2 system)
Rearranged index/TOC to follow installation chronological order
Added cross-references to other chapters from Quickstart
Added terms to Glossary

### TESTS CONDUCTED:
None (N/A)

### DEPENDENCIES:
None known (N/A)

### ISSUE:
Makes the documentation more intuitive and readable for beginners. 
Resolves the lack of HPC-Stack Install documentation in the SRW docs. 

### CONTRIBUTOR:
@kb1kgp

### CODE REVIEW

File: docs/UsersGuide/source/ConfigNewPlatform.rst [hunk 1]
```diff
@@ -275,7 +275,7 @@ The ``README.md`` file will contain instructions on the order that each script s
    +--------------------+----------------------------+------------+-----------+
    | ufs-weather-model  | ./run_fcst.sh              | 6          | 1h 40 m   |
    +--------------------+----------------------------+------------+-----------+
-   | UPP                | ./run_post.sh              | 1          | 7 m       |
+   | EMC_post           | ./run_post.sh              | 1          | 7 m       |
    +--------------------+----------------------------+------------+-----------+
```
Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/ConfigNewPlatform.rst]: This needs to stay UPP.  It was recently changed and updated in the umbrella repo.

File: docs/UsersGuide/source/CodeReposAndDirs.rst [hunk 2]
```diff
@@ -30,14 +30,14 @@ repositories associated with this umbrella repo (see :numref:`Table %s <top_leve
    | Repository for                  | https://github.com/ufs-community/ufs-weather-model      |
```
Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/CodeReposAndDirs.rst]: This file shouldn't change, as it was updated recently to account for the new GitHub organization used for regional_workflow and the name change for UPP.

File: docs/UsersGuide/source/ConfigParameters.inc [hunk 3]
```diff
@@ -310,7 +310,7 @@ Maximum number of attempts.
 Customized Post Configuration Parameters
 ========================================
 ``USE_CUSTOM_POST_CONFIG_FILE``: (Default: “FALSE”)
-   Flag that determines whether a user-provided custom configuration file should be used for post-processing the model data. If this is set to "TRUE", then the workflow will use the custom post-processing (UPP) configuration file specified in ``CUSTOM_POST_CONFIG_FP``. Otherwise, a default configuration file provided in the UPP repository will be used.
+   Flag that determines whether a user-provided custom configuration file should be used for post-processing the model data. If this is set to "TRUE", then the workflow will use the custom post-processing (UPP) configuration file specified in ``CUSTOM_POST_CONFIG_FP``. Otherwise, a default configuration file provided in the EMC_post repository will be used.
 
 ``CUSTOM_POST_CONFIG_FP``: (Default: “”)
    The full path to the custom post flat file, including filename, to be used for post-processing. This is only used if ``CUSTOM_POST_CONFIG_FILE`` is set to "TRUE".
```
Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/ConfigParameters.inc]: This file shouldn't change (needs to stay as "UPP").

**Review by JeffBeck-NOAA — CHANGES_REQUESTED** [review]
Gillian, thanks for opening this draft PR.  Could you use the official PR template that you see in other PR conversation tabs? It should have auto-populated when you opened the PR. The template format helps to identify changes, tests run, documentation changes, etc.  The PR title also states that this is to "update fork", which I'm assuming you didn't mean?  Also, please remove the ".DS_Store" binary files from the PR.  Thanks!

**Comment by gspetro-NOAA** [comment]
Will do! Didn't realize you could see the draft... It's definitely still a
draft as the Introduction is only half done, it's got comments all over,
and we haven't finished running the SRW on all systems!

Best,
Gillian

**Review by ligiabernardet — COMMENTED** [review]
I know this is just a draft, but I thought I'd leave a comment. It is great to have HPC Stack documentation. I recommend coming up with a mechanism to make it usable for other Apps and also for standalone installation. For example: create some a standalone rst file and include it in the SRW App User's Guide.

**Comment by JeffBeck-NOAA** [comment]
> Will do! Didn't realize you could see the draft... It's definitely still a draft as the Introduction is only half done, it's got comments all over, and we haven't finished running the SRW on all systems! Best, Gillian

Thanks, Gillian!  Looking forward to getting this incorporated into the repo.  Thank you for your work on the documentation!

File: docs/UsersGuide/source/InputOutputFiles.rst [hunk 4]
```diff
@@ -200,7 +200,7 @@ the user in the ``config.sh`` settings.
 If you wish to modify the fields or levels that are output from the UPP, you will need to make
 modifications to file ``fv3lam.xml``, which resides in the UPP repository distributed with the UFS SRW
 Application. Specifically, if the code was cloned in the directory ``ufs-srweather-app``, the file will be
-located in ``ufs-srweather-app/src/UPP/parm``.
+located in ``ufs-srweather-app/src/EMC_post/parm``.
```
Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/InputOutputFiles.rst]: Want to keep this UPP, thanks!

File: docs/UsersGuide/source/InputOutputFiles.rst [hunk 5]
```diff
@@ -200,7 +200,7 @@ the user in the ``config.sh`` settings.
...
```
Comment (mkavulich) [inline docs/UsersGuide/source/InputOutputFiles.rst]: FYI the official name is now UPP, EMC_post was used for the last release

**Pull request closed without merging**